CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+
+## Issues
+
+Issues are a great way to keep track of tasks, enhancements, and bugs for projects. If you find a bug, or have a suggestion for enhacement, feel free to open a issue. You can open one using the green button "New issue", located in the "Issues" tab or the project.
+
+### What a a great issue has
+
+So what do you need to fill in a issue:
+ * Title: The title should describe the bug or enhacement briefly.
+ * Labels: Check if the project has labels and add the labels that describes the issue.
+ * Summary: Summarize the bug/enhacement concisely
+ * Steps to reproduce (Bugs only): How the developer or others users can reproduce the bug?
+ * Revelant information: Add logs, or screenshots if filling a bug, or maybe a mockup if it's a enhacement.
+ * Enviroment info: Tell the version of software and OS you encountered the issue. 
+
+Tip: Check first if a similar issue hasn't filled before!
+
+## Pull request
+
+Pull requests (PRs) let you tell others about changes you've pushed to a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before the changes are merged into the repository.
+
+Follow the contribution etiquette of the project located in CONTRIBUTING.md.
+
+In order to create a PR, you can use the white button located in the main page of the project you want to contribute. Note that maybe you need to create a fork frist so you can work on the changes before making the PR. [More info here](https://help.github.com/articles/creating-a-pull-request-from-a-fork/). You PR should also use a branching strategy, noted down.
+
+### What a great PR has
+
+ * Title: Should describe what the PR does.
+ * Summary: A description. You usually should only address one problem per PR.
+ * Tags: Use the tags the project have to help the reviewer(s).
+ * Issue: What issues are addressed on this PR?
+ * Tests: If the project demands tests for new code, be sure to add it. Also, make sure the code builds and pass all the tests, before sending the PR.
+ * Don't litter: Follow the coding principles of the project. Squash commits that doesn't add anything by themselves.
+ * Wait for feedback: If the reviewer asks for changes, be sure to address them, otherwise, your PR might be closed.
+
+## Branching strategy
+
+Every project follows a branching strategy, that means, some rules on what each branch name means, and how to name them.
+
+One of the most used branching strategy in Github is [GitHub Flow](https://guides.github.com/introduction/flow/), where you create a branch from the default branch (usually 'master'), put your changes, and the send a PR for discussion. That way, your changes can be reviewed before being accepted.
+
+As a rule of thumb, don't add new changes to 'master', even if the project you are contributing doesn't specify which branching strategy is using.
+
+Happy hacking!
```

Comment:
Happy Open Sourcing maybe (just a suggestion) 😃

Reply:
Cool! I will add that